### PR TITLE
Filemerger test fixes

### PR DESCRIPTION
**Pull request by oshadura**
Investigation doesn't show any data differences in histograms, comparison shows that for TH1F/TH2F was added extra bytes, not affecting data of histograms. Hsimple101.root reference was updated according hsimple.root (with hadd -f1 option and default ZLIB algorithm)

### CODE REVIEW

File: root/io/filemerger/execTestMultiMerge.C [hunk 1]
```diff
@@ -70,19 +70,22 @@ int testSimpleFile(const char *filename, Long64_t entries, Int_t compSetting, Lo
 int execTestMultiMerge()
 {
    Int_t result = 0;
-   int hsimpleFTolerance = 6; // 5 for non fst builds
+   int hsimpleFTolerance = 10; // 5 for non fst builds
```
Comment (pcanal) [inline root/io/filemerger/execTestMultiMerge.C]: Do we have a way to detect if were are in a 'fst' build?

Comment (oshadura) [reply root/io/filemerger/execTestMultiMerge.C]: @pcanal it could defined R__FAST_MATH, I will push new version of fix

Comment (oshadura) [reply root/io/filemerger/execTestMultiMerge.C]: @pcanal sorry, I mistaken hsimpleFTolerance is for hsimpleF.test case (it is failing only on not-fast aarch64 native compiler build (gcc4.8.5))

**Comment by oshadura** [comment]
@pcanal, I introduced as you ask separation of fast, i686 builds and others.Can you look please?

**Pull request merged**